Difi.Oppslagstjeneste.Klient/OppslagstjenesteKlient.cs

```diff
@@ -20,7 +20,7 @@ namespace Difi.Oppslagstjeneste.Klient
     public class OppslagstjenesteKlient
     {
         private static readonly ILog Log = LogManager.GetLogger(MethodBase.GetCurrentMethod().DeclaringType);
-        private static readonly ILog RequestLog = LogManager.GetLogger($"{typeof (OppslagstjenesteKlient).Namespace}.RequestLog");
+        private static readonly ILog RequestLog = LogManager.GetLogger("Difi.Oppslagstjeneste.RequestLog");
```

Comment:
Hvorfor ble denne endret? Nå som vi har en bool for å aktivere loggingen, så er det vel nok med å ha det slik som det var?  `Difi.Oppslagstjeneste.Klient` bør fly bra?